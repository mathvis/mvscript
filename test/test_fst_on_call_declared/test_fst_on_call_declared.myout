Decl (FunctionDef (FunctionIdentifier "a") [] VoidT (Just (Block (FunctionBlock VoidT) [])))
Expr (FunctionCall (FunctionIdentifier "a") [])
