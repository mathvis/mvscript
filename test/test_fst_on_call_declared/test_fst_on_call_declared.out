Decl (FunctionDef (FunctionIdentifier "a") [] void (Just (Block (FunctionBlock void) [])))
Expr (FunctionCall (FunctionIdentifier "a") [])
